component/alert/alert.css

```diff
@@ -0,0 +1,59 @@
+.alert-heading {
+    font-size: 2rem;
+    margin-top: 40px;
+  }
+  .code-snippet-link {
+    box-sizing: border-box;
+    max-height: 400px;
+    width: 100%;
+    overflow: auto;
+    padding: 1rem 0;
+    margin-bottom: 2rem;
+  }
+  
+  .code-example {
+    margin:2rem 0 1rem 0;
+    width: 95%;
+    max-width: 1000px;
+    height: 200px;
+  }
+  
+  .pink{
+    background-color:var(--light-pink);
+  }
+  .styling-color{
+    color:var(--light-pink);
+  }
+  .alert-common {
+    padding: 1rem 0.5rem;
+    margin-bottom: 1rem;
+    border-radius: 0.3rem;
+  }
+  .alert-info {
+    border: 2px solid var(--dark-blue);
+    background-color: var(--sky-blue);
+    color: var(--dark-blue);
+    display: flex;
+    justify-content: space-between;
```

Comment:
Please remove display:flex as it is unused.

Reply:
done!!